src/main/webapp/groups.js

```diff
@@ -26,19 +26,18 @@ function makeCalendarId() {
     const name = document.getElementById('name').value;
     const image = document.getElementById('image').value;
     const description = document.getElementById('description-area').value;
-    let newGroupId;
+    let newGroupId = "";
```

Comment:
if you don't use newGroupId anywhere else in the method, you can just delete this line